### ISSUE TITLE
breaks with single page citations

### ISSUE THREAD

**Issue by dcherian** [andras-simonyi/citeproc-orgref#1]
Example bibtex entry
```
@article{Dyson2004,
  title = {A Meeting with {{Enrico Fermi}}},
  volume = {427},
  issn = {0028-0836, 1476-4679},
  doi = {10.1038/427297a},
  number = {6972},
  journal = {Nature},
  author = {Dyson, Freeman},
  month = jan,
  year = {2004},
  pages = {297--297},}
```

`citeproc-prange--end-significant` throws an error saying `Wrong argument`

**Comment by andras-simonyi** [comment]
@dcherian, thanks for reporting this -- I haven't encountered such a "degenerate" page range before. The only reasonable behavior I can imagine is to automatically reduce these to a one page locator (as opposed to a range). Do you see  situations where this would give an incorrect result?

**Comment by dcherian** [comment]
Yes single page number seems to be how people cite it.

**Comment by andras-simonyi** [comment]
This issue was moved to andras-simonyi/citeproc-el#3

**Comment by andras-simonyi** [comment]
Since it was caused by the underlying citeproc-el library, I've moved the issue and fixed it there -- [release 0.1.1](https://github.com/andras-simonyi/citeproc-el/releases/download/0.1.1/citeproc-0.1.1.tar), which I've just released already contains the fix.

**Comment by dcherian** [comment]
Awesome thanks.
